scripts/setup.sh

```diff
@@ -15,13 +15,17 @@
 # limitations under the License.
 
 K8S_VERSION=$1
+KIND_NAME="kind"
+if [ $# -gt 1 ]; then
+    KIND_NAME=$2
+fi
 
```

Comment:
Is this change to address to allow we run e2e locally, Am I right? Could we do a specific PR to address this need?